eth/beacon/state_machines/forks/serenity/state_transitions.py

```diff
@@ -40,7 +43,15 @@ def per_slot_transition(self,
 
     def per_block_transition(self, state: BeaconState, block: BaseBeaconBlock) -> BeaconState:
         # TODO
```

Comment:
Is this `TODO` a leftover?

Reply:
Mostly just that anyone reading the code will have no context for what is left `TODO`..